packages/contracts/contracts/voting/admin-address/AdminAddress.sol

```diff
@@ -0,0 +1,71 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import {Counters} from "@openzeppelin/contracts/utils/Counters.sol";
+
+import {PluginCloneable} from "../../core/plugin/PluginCloneable.sol";
+import {IDAO} from "../../core/IDAO.sol";
+
+/// @title AdminAddress
+/// @author Aragon Association - 2022.
+/// @notice The Admin address governance plugin, to serve as a DAO's primary full controller.
+contract AdminAddress is PluginCloneable {
```

Comment:
By convention, I would always name these ontracts like `xxxPlugin` and `xxxPluginSetup`

In this case I would do `AdminPlugin`

Reply:
Would AdminGovernance be a better name?
If the name "Governor" would not be taken already by OZ, I would use it here.